src/components/Form/Fields/Select.js

```diff
@@ -1,19 +1,25 @@
 import React, { PureComponent } from 'react';
+import ReactSelect, { components } from 'react-select';
```

Comment:
Not keen on `components` being imported into the namespace, it's not very descriptive and quite similar to React's own `Component`.

I think this might be better as:
```
import ReactSelect from 'react-select';
import { Option } from 'react-select/lib/components';
```

or:

```
import ReactSelect, { components as ReactSelectComponents } from 'react-select';
const { Option } = ReactSelectComponents;
```

Reply:
Agreed.